atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1484,6 +1486,32 @@ public void commitThrowsIfRolledBackAtCommitTime_expiredLocks() {
                 .hasLocksExpired(1);
     }
 
+    @Test
+    public void commitThrowsIfLockAcquisitionFails() {
+        final Cell cell = Cell.create(PtBytes.toBytes("row1"), PtBytes.toBytes("column1"));
+
+        TimelockService timelockServiceMock = mock(TimelockService.class);
```

Comment:
(Nothing to change here, but for your own info: We don't tend to explicitly denote a mock in the variable name generally, although we seem to do it more in this test)